src/styles/_base.css

```diff
@@ -2,6 +2,10 @@
   box-sizing: border-box;
 }
 
+html {
+  height: -webkit-fill-available;
```

Comment:
Do you mind explaining that in a comment? I don't know, what it does here.

Reply:
Please try it the way I shared with you in [this codepen](https://codepen.io/neuefische/pen/PoZeqzQ?editors=0100).